Fixed bug in FreeBSD cache prune operation

cdf-eagles opened the pull request:
The path for the CACHE variable was incorrect for pruning operations.

This has been corrected and tested.

`Select

  1 - 'stop'      - Stop PMS.
  2 - 'automatic' - Check, Repair/Optimize, and Reindex Database in one step.
  3 - 'check'     - Perform integrity check of database.
  4 - 'vacuum'    - Remove empty space from database without optimizing.
  5 - 'repair'    - Repair/Optimize databases.
  6 - 'reindex'   - Rebuild database indexes.
  7 - 'start'     - Start PMS

  8 - 'import'    - Import watch history from another database independent of Plex. (risky).
  9 - 'replace'   - Replace current databases with newest usable backup copy (interactive).
 10 - 'show'      - Show logfile.
 11 - 'status'    - Report status of PMS (run-state and databases).
 12 - 'undo'      - Undo last successful command.

 21 - 'prune'     - Remove old image files (jpeg,jpg,png) from PhotoTranscoder cache & all temp files left by PMS.
 42 - 'ignore'    - Ignore duplicate/constraint errors.

 88 - 'update'    - Check for updates.
 98 - 'quit'      - Quit immediately.  Keep all temporary files.
 99 - 'exit'      - Exit with cleanup options.

Enter command # -or- command name (4 char min) : 21
 
Counting how many files can be removed.
OK to prune 1712 files?  (Y/N) ? n
`

ChuckPa commented:
Sorry to have bothered.  (rough day here)

I dug in and fixed it myself.

I'm running it through its paces and preparing the pull which merges your work and updated documentation into the formal release of 1.11.07

Thanks for all your work.


PS:   It's perfectly OK to prune those files.    They're 30 days old (default age).   If PMS needs them again,  it will recreate them.     Prune is really useful for large libraries  (I've seen some with 25,000+ files ,  all pics,  pruned)

cdf-eagles commented:
No worries. Happy to help! The script has saved my library more than once, so I figured I’d give my changes a more formal contribution, since I run it on FreeBSD at home.The only reason I didn’t execute the prune was because I was actively watching something on Plex and wanted to get the service restarted as quick as possible to prevent an interruption.—Sent from my mobile deviceOn Jun 25, 2025, at 19:11, Chuck ***@***.***> wrote:﻿ChuckPa left a comment (ChuckPa/DBRepair#227)
Sorry to have bothered.  (rough day here)
I dug in and fixed it myself.
I'm running it through its paces and preparing the pull which merges your work and updated documentation into the formal release of 1.11.07
Thanks for all your work.
PS:   It's perfectly OK to prune those files.    They're 30 days old (default age).   If PMS needs them again,  it will recreate them.     Prune is really useful for large libraries  (I've seen some with 25,000+ files ,  all pics,  pruned)

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

cdf-eagles commented:
@ChuckPa FYI, in your branch freebsd-releases, you missed one Cache path update.

cdf-eagles commented:
Closing this PR since the changes are redundant.
